api/management/README.md

```diff
@@ -0,0 +1,125 @@
+# Services management
+
+The API described here are intended to assist with the operation and management
+of a Veraison services deployment. They are not meant to be utilized by the
+services' users.
+
+## Policy management
+
+Policies allow augmenting attestation scheme generated results on
+per-deployment basis, by specifying additional rules to be applied to these
+results. This may result in the change of the trust vector values (and,
+correspondingly, the overall result status), and/or addition of supplementary
+claims under the `policy-added-claims` EAR extension.
+
+A policy is defined by a text document describing the rules to be applied by
+the policy engine. Currently, Veraison only support the OPA policy engine.
+Correspondingly, policies must be written in Rego language used by OPA. See
+[OPA policy README] in services repo for details.
+
+Policies are automatically retrieved, based on the attestation scheme and
+tenant id for the evidence being evaluated. Currently, only one policy is
+allowed per tenant/scheme combination. If a policy is added for a tenant/scheme
```

Comment:
For a scheme, if a tenant changes the policy should change ? So should tenant not be part of the API ?

I understand, for now, we have not used tenant in the API, if we want to retain the same, please add the required clarification in the interface, as `tenancy` aspect has been explicitly highlighted in the interface description

Reply:
My underestanding is that tenant id is intended to be internal and will be resolved based on authentication (i.e. an account is associated with a specific tenant). It will not be known a priori by the management actor. Authenticaion is orthongonal to this interface, so it doesn't need to be tied to a tenant.